docs/storage/lumio/advanced.md

```diff
@@ -1,138 +1,318 @@
-# Advanced usage of LUMI-O  
+# Sharing access to data
 
-## Introduction
+Sharing data, e.g. datasets or research results, is (in principle) easy in the object storage. One should pay attention to understand what they are doing, though, especially before changing access rights to the data (with ACLs or Policies), or to not unintendedly publish something that shouldn't be publicly accessible.
```

Comment:
I would try to make this and next paragraphs more compact, there is too much general text here. Emphasise more on different sharing options.

Reply:
I tried to reduce the amount of "introductory" text on this page. Deleted some (that was maybe somewhat repetition) and moved some..